priv/repo/migrations/20191117042320_add_cost_field_to_charges.exs

```diff
@@ -0,0 +1,11 @@
+defmodule TeslaMate.Repo.Migrations.AddCostFieldToCharges do
+  use Ecto.Migration
+
+  def change do
+
+    alter table(:charges) do
```

Comment:
The cost field should be added to the `:charging_processes` table. The naming could be better, I know :-)